criteria.c

```diff
@@ -134,6 +135,9 @@ bool parse_criteria(const char *string, struct mako_criteria *criteria) {
 				++token_location;
 			}
 			break;
+		default:
```

Comment:
Hmm. I'm not a fan of adding `default` cases since most compilers will emit a warning when an enum case isn't handled.